### ISSUE TITLE
Nuke Ops should have a different way of picking their class

### ISSUE THREAD

**Issue by agentofzog2** [d3athrow/vgstation13#23472]
It should be done via a machine so nuke ops can pick the classes for whatever approach they're trying to do instead of picking something right as they spawn,
I guess you don't have to pick a class (unless it interferes with the commanders name picking) but the menu still encourages you to pick it early.

**Comment by Diavill** [comment]
have class preference be a thing when having nug obs on so you already get the class you want and in the case of conflicts have the conflicting parties fight to the death or some shit

**Comment by DeityLink** [comment]
Agreed, their base should be reworked to be kinda like the ERT's, with their suit dispenser letting them choose their class.

**Comment by agentofzog2** [comment]
fucking late but it does interfere with the commander's name picking
